### PR TITLE
Update docs: mention server-side rate-limiting

### PR DESCRIPTION
**Pull request by foglcz**
Also updated in Matej documentation.

This feature is coming into Matej v7.39.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,12 @@ $response = $matej->request()
     ->send();
 ```
 
+**This endpoint has rate-limiting implemnted.** We constantly monitor workload on backend systems,
```
Comment (OndraM) [inline README.md]: implemented (typo)

File: README.md [hunk 2]
```diff
@@ -156,6 +156,12 @@ $response = $matej->request()
...
+and when the # of events in the queue crosses certain treshold, Matej API will start returning `503` errors.
```
Comment (OndraM) [inline README.md]: threshold (typo)

Comment (OndraM) [inline README.md]: I'd replace "#"  with "number" (for clarity)

**Review by OndraM — CHANGES_REQUESTED** [review]

**Comment by OndraM** [comment]
Ping @foglcz ?

**Comment by foglcz** [comment]
Fixed typos, thanks for the ping @OndraM :)

**Review by OndraM — APPROVED** [review]

**Pull request merged**